projects/npm-tools/packages/npm-scripts/src/utils/parseBnd.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * SPDX-FileCopyrightText: © 2019 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+const fs = require('fs');
+
+/**
+ *
+ */
+function parseBnd(filePath = './bnd.bnd') {
+	const content = fs.readFileSync(filePath).toString();
+	const lines = content.replace(/\\\n/g, ' ').split('\n');
+
+	return lines.reduce((entries, line) => {
+		line = line.trim();
+
+		if (line === '') {
+			return entries;
+		}
+
+		const i = line.indexOf(':');
+		const key = line.substring(0, i);
+		const value = line.substring(i + 2);
```

Comment:
I know we expect whitespace after the `:` and source formatter will require it, but what if we ever run on an unformatted file? Shouldn't we avoid hard-coding the `+ 2` here?

Reply:
:thinking: Not sure. I thought it was a requirement (at least it is in the final `MANIFEST.MF` AFAIK). But it is probable that `bnd` doesn't complain if it is missing, so I will trim it instead, just to be safe :+1: